test/client/filters.clienttest.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  'Filtering on search page': function (browser) {
+    browser
+      .url('http://localhost:8000/search?q=charles')
```

Comment:
Yeah we're running it on travis via saucelabs, so we can see the videos there. I do want to do a bit more with nightwatch this week as part of #162, so I'll probably save some screenshots and add more browsers